<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QPushButton" name="otchButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="text">
      <string>Отчеты</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="8">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>5</number>
     </property>
     <widget class="ActTab" name="ActWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="PredTab" name="PredWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="ZayTab" name="ZayWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="ClientTab" name="ClientWidget"/>
     <widget class="OtchTab" name="OtchWidget"/>
     <widget class="NastrTab" name="NastrWidget"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="commButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="text">
      <string>Коммерческие
предложения</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="zayButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="text">
      <string>Заявки</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>93</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="nastrButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="text">
      <string>Настройки</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="actButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="text">
      <string>Акты</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="clientButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="text">
      <string>Клиенты</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActTab</class>
   <extends>QWidget</extends>
   <header>actwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZayTab</class>
   <extends>QWidget</extends>
   <header>zaywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PredTab</class>
   <extends>QWidget</extends>
   <header>predwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClientTab</class>
   <extends>QWidget</extends>
   <header>clientwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OtchTab</class>
   <extends>QWidget</extends>
   <header>otchwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NastrTab</class>
   <extends>QWidget</extends>
   <header>nastrwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
